frontend/src/components/template/TheHeader.vue

```diff
@@ -4,7 +4,7 @@
       <section class="container">
         <section class="identity">
           <img src="../../assets/images/logo.png" width="154" class="logo" alt="BC Government Logo" />
-          <div class="sitename">MoH User Management</div>
+          <div class="sitename">{{ title }}</div>
```

Comment:
This is great for the `sitename` but the `<head><title>` is still MoH User Management.

Reply:
fixed